src/ImageRunModal.jsx

```diff
@@ -463,13 +463,19 @@ export class ImageRunModal extends React.Component {
     }
 
     clearImageSelection = () => {
+        // Reset command if it was prefilled
+        let command = this.state.command;
+        if (this.state.selectedImage && this.state.selectedImage.Command && this.state.command === utils.quote_cmdline(this.state.selectedImage.Command))
+            command = "";
+
         this.setState({
             selectedImage: "",
             image: "",
             isImageSelectOpen: false,
             imageResults: {},
             searchText: "",
             searchFinished: false,
+            command: command,
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1155-20221207-145112-86329344-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)